numpy/f2py/src/fortranobject.c

```diff
@@ -986,47 +1071,56 @@ int check_and_fix_dimensions(const PyArrayObject* arr, const int rank, npy_intp
                          new_size, arr_size);
             return 1;
         }
-    } else { /* [[1,2]] -> [[1],[2]] */
-        int i,j;
+    }
+    else { /* [[1,2]] -> [[1],[2]] */
+        int i, j;
         npy_intp d;
         int effrank;
         npy_intp size;
-        for (i=0,effrank=0;i<PyArray_NDIM(arr);++i)
-            if (PyArray_DIM(arr,i)>1) ++effrank;
-        if (dims[rank-1]>=0)
-            if (effrank>rank) {
+        for (i = 0, effrank = 0; i < PyArray_NDIM(arr); ++i)
+            if (PyArray_DIM(arr, i) > 1)
+                ++effrank;
```

Comment:
Hmm, I thought single line statements like this would get brackets. Might need to check .clang-format.

Reply:
Nope, not with what [we have now](https://clang.llvm.org/docs/ClangFormatStyleOptions.html), which is `BreakBeforeBraces: Stroustrup`.

`clang-tidy` has `-checks="readability-braces-around-statements"` which would do that.

Reply:
Seems there is no way to control that yet.

Reply:
Google style allows this if there is no `else` or `else if` in the conditional.

Reply:
I added a commit for `clang-tidy` and perhaps we should consider a companion `.clang-tidy` for enforcing this.